worlds/sm64ex/Options.py

```diff
@@ -127,6 +127,30 @@ class MoveRandomizerActions(OptionSet):
     valid_keys = [action for action in action_item_table if action != 'Double Jump']
     default = valid_keys
 
+sm64_options_groups = [
+    OptionGroup("Logic Options", [
```

Comment:
Now that this is called logic options, `AreaRandomizer` should be part of this

Reply:
Added. Was initially in its own category called "Level Shuffling" (SMW).